File: typescript/plushie_pad/experiments/hello.js
// Experiment: hello
// The last expression in this file is the rendered view node.
// The `ui` binding is injected automatically; no imports needed.

ui.column({ padding: 16, spacing: 8, id: "root" }, [
  ui.text("Hello from hello!", { id: "greeting", size: 24 }),
  ui.text("Edit me and press Save.", { id: "hint" }),
])

File: ruby/plushie_pad/experiments/hello.rb
# Experiment: hello
# Define Experiment.view returning a Plushie node.
module Experiment
  def self.view
    Plushie::Widget::Column.new("root", padding: 16, spacing: 8)
      .push(Plushie::Widget::Text.new("greeting", "Hello from hello!", size: 24))
      .push(Plushie::Widget::Text.new("hint", "Edit me and press Save."))
      .build
  end
end
